### PR TITLE
chore: logs and tests clean-up

### PR DESCRIPTION
**Pull request by patrickzheng200**
This PR targets on issues #251, #252 and #254.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@
 *.sublime-workspace
 
 # Custom
-coverage.txt
\ No newline at end of file
+coverage.txt
+tmp
```
Comment (JeyJeyGao) [inline .gitignore:9]: Why we need it?

Comment (patrickzheng200) [reply .gitignore:9]: Running `go test -v .` at the root dir would generate a folder tmp for trust store due to the example_test files.

**Review by JeyJeyGao — APPROVED** [review]
LGTM

File: .gitignore (hunk 1 above)
Comment (priteshbandi) [reply .gitignore:9]: Shouldn't we be cleaning that file after test run?

File: verifier/helpers.go [hunk 2]
```diff
@@ -55,7 +55,7 @@ func loadX509TrustStores(ctx context.Context, scheme signature.SigningScheme, po
 
 		storeType, name, found := strings.Cut(trustStore, ":")
 		if !found {
-			return nil, fmt.Errorf("trust policy statement %q is missing separator in trust store value %q", policy.Name, trustStore)
+			return nil, fmt.Errorf("trust policy statement %q is missing separator in trust store value %q. Format <TrustStoreType>:<TrustStoreName> is required", policy.Name, trustStore)
```
Comment (priteshbandi) [inline verifier/helpers.go:58]: nit: Should we reword it to:
`trust policy statement %q is missing separator in trust store value %q. The required format is <TrustStoreType>:<TrustStoreName>`

**Review by priteshbandi — APPROVED** [review]
LGTM with a nit

**Review by shizhMSFT — APPROVED** [review]
LGTM
